### PR TITLE
Add blackbox-terminal

### PR DESCRIPTION
**Pull request by wszqkzqk**
一个基于VTE4/Vala的很漂亮的终端，参见[其GNOME GitLab页面](https://gitlab.gnome.org/raggesilver/blackbox)

### CODE REVIEW

File: archlinuxcn/blackbox-terminal/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Maintainer:  wszqkzqk <wszqkzqk@qq.com>
+# Author:  Dimitris Kiziridis <dkiziridis at outlook dot com>
+
+pkgname=blackbox-terminal
+pkgver=0.13.2
+pkgrel=1
+pkgdesc='A beautiful GTK 4 terminal'
+arch=('i686' 'x86_64')
```
Comment (hubutui) [inline archlinuxcn/blackbox-terminal/PKGBUILD:8]: ArchLinux still has `i686`?

Comment (wszqkzqk) [reply archlinuxcn/blackbox-terminal/PKGBUILD:8]: It's unofficially supported.

File: archlinuxcn/blackbox-terminal/lilac.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    update_pkgver_and_pkgrel(_G.newver)
+
+post_build_script: |
+    git_pkgbuild_commit()
+
+update_on:
+  - source: gitlab
+    gitlab: raggesilver/blackbox
+    host: gitlab.gnome.org
+    use_latest_release: true
```
Comment (hubutui) [inline archlinuxcn/blackbox-terminal/lilac.yaml:17]: There is no `use_latest_release` config for gitlab, check also the [doc](https://nvchecker.readthedocs.io/en/latest/usage.html#check-gitlab) of nvchecker.

**Comment by hubutui** [comment]
make sure you could build the pkg in a clean chroot, see also this [wiki](https://wiki.archlinux.org/title/DeveloperWiki:Building_in_a_clean_chroot).

**Comment by SilverRainZ** [comment]
Any progress?

**Comment by wszqkzqk** [comment]
> Any progress?

Sorry, I finally chose to use konsole and now have no drive to port it.

**Pull request closed without merging**